src/component/plotly-dash-preview/parse.js

```diff
@@ -31,6 +32,19 @@ function parse (body, opts, sendToRenderer) {
   result.loadingSelector = body.loading_selector
   result.timeOut = body.timeout
 
+  if (cst.sizeMapping[result.pdfOptions.pageSize]) {
+    result.browserSize = cst.sizeMapping[result.pdfOptions.pageSize]
+  } else if (body.pageSize && body.pageSize.width && body.pageSize.height) {
```

Comment:
Use `isPositiveNumeric(body.pageSize.width)` and `isPositiveNumeric(body.pageSize.height)` to error early on 0 and negative inputs.